cli/src/main.rs

```diff
@@ -8,9 +8,9 @@ struct Args {
     /// The type of file is determined from the file extension.
     #[arg(short = 'm', long = "manifest", value_name = "FILE")]
     manifest_path: PathBuf,
-    /// Path to output location. Any existing file is overwritten.
+    /// Path to output location. Any existing file is overwritten. Defaults to stdout.
```

Comment:
This should probably say: "If not provided, the output is directed to the stdout".
Now it suggests you can use streams as the value too. (Maybe that's true on linux even through the fs API, don't know, but likely not on e.g. windows)